modules/msk/genomenexus/annotationpipeline/meta.yml

```diff
@@ -0,0 +1,52 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "genomenexus_annotationpipeline"
+
+description: DSL2 module to perform annotation of a MAF file type using the genomenexus suite of conversion and annotation tools.
+keywords:
+  - annotate
+  - vcf
+  - maf
+  - genomenexus
+  - genomics
+tools:
+  - "genomenexus":
+      description: "DSL2 module to perform an annotation of MAF file type using the genomenexus suite of conversion and annotation tools."
+      homepage: "https://github.com/genome-nexus/genome-nexus-annotation-pipeline/"
+      documentation: "https://github.com/genome-nexus/.github/blob/main/profile/README.md"
```

Comment:
I would update the description. The tool itself (genomenexus) is not a nextflow dsl2 module its just a java pipeline.
Also: you can use this for the README, the .github link might not be stable